packages/html/src/signature/tests/signature.tsx

```diff
@@ -1,457 +1,71 @@
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="window-maximize"
-                    className="k-signature-action k-signature-minimize"
-                >
-                </Button>
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="window-minimize"
-                    className="k-signature-action k-signature-minimize k-hidden"
-                >
-                </Button>
+            <div dir="rtl">
+                <Signature invalid focus />
             </div>
-            <canvas className="k-signature-canvas">
-                An alternate text describing what the canvas displays.
-            </canvas>
-            <div className="k-signature-line"></div>
-            <div className="k-signature-actions k-signature-actions-bottom">
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="close"
-                    className="k-signature-action k-signature-clear"
-                >
-                </Button>
+
+            <div>
+                <Signature disabled />
             </div>
-        </div>
-        <div className="k-signature k-signature-md k-input k-input-solid k-rounded-md k-signature-maximized">
-            <div className="k-signature-actions k-signature-actions-top">
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="window-maximize"
-                    className="k-signature-action k-signature-minimize"
-                >
-                </Button>
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="window-minimize"
-                    className="k-signature-action k-signature-minimize k-hidden"
-                >
-                </Button>
+            <div dir="rtl">
+                <Signature disabled />
             </div>
-            <canvas className="k-signature-canvas">
-                An alternate text describing what the canvas displays.
-            </canvas>
-            <div className="k-signature-line"></div>
-            <div className="k-signature-actions k-signature-actions-bottom">
-                <Button
-                    fillMode="flat"
-                    size="medium"
-                    rounded="medium"
-                    icon="close"
-                    className="k-signature-action k-signature-clear"
-                >
-                </Button>
+
+            <div className="k-colspan-all k-col-span-full">
+                <Signature className="k-signature-maximized" />
```

Comment:
Add a `maximized` prop to the component that adds the class conditionally e.g:

<Signature maximized />